### PR TITLE
DOR cleaning 

### PR DESCRIPTION
**Pull request by kprokofi**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -14,8 +14,8 @@ terminaltables
 pytorchcv
 torch-lr-finder==0.2.1
 onnx
-torchvision==0.9.*
-torch==1.8.*
```
Comment (sovrasov) [inline requirements.txt]: This change should be in line with the SC deps update schedule. I don't familiar with any plans to update pytorch there.

File: torchreid/data/transforms.py [hunk 2]
```diff
@@ -756,250 +530,78 @@ def __init__(self, p, sigma, grayscale):
...
-        return F.normalize(image, self.mean, self.std, self.inplace), mask
-
+    def __call__(self, image):
+        return F.normalize(image, self.mean, self.std, self.inplace)
 
-class PairToTensor(object):
-    def __call__(self, input_tuple):
-        image, mask = input_tuple
 
+class ToTensor(object):
```
Comment (sovrasov) [inline torchreid/data/transforms.py]: Seems like torchvision's ToTensor now should work here.

File: torchreid/integration/sc/inference_task.py [hunk 3]
```diff
@@ -154,15 +154,13 @@ def _patch_config(self, base_dir: str):
             config_file_path = os.path.join(base_dir, 'main_model_multilabel.yaml')
         else:
             config_file_path = os.path.join(base_dir, 'main_model.yaml')
+        print(self._cfg.train.keys())
```
Comment (sovrasov) [inline torchreid/integration/sc/inference_task.py]: Pls del debug print

File: torchreid/engine/image/am_softmax.py [hunk 4]
```diff
@@ -7,30 +7,29 @@
 
 from __future__ import absolute_import, division, print_function
 import copy
+from importlib.metadata import requires
 
 import numpy as np
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
+from torch.autograd import Variable
```
Comment (sovrasov) [inline torchreid/engine/image/am_softmax.py]: Seems like this import is unused

File: torchreid/engine/image/am_softmax.py [hunk 5]
```diff
@@ -7,30 +7,29 @@
 
 from __future__ import absolute_import, division, print_function
 import copy
+from importlib.metadata import requires
```
Comment (sovrasov) [inline torchreid/engine/image/am_softmax.py]: Seems like this import is unused

File: torchreid/engine/image/am_softmax.py [hunk 6]
```diff
@@ -73,181 +68,93 @@ def __init__(self, datamanager, models, optimizers, reg_cfg, metric_cfg, schedul
...
 
-                        mutual_loss += m_loss
-                        loss_summary['mutual_{}/{}'.format(trg_id, model_names[model_id])] = m_loss.item()
-                        num_mutual_losses += 1
+                    should_turn_off_mutual_learning = self._should_turn_off_mutual_learning(self.epoch)
+                    coeff_mutual_learning = int(not should_turn_off_mutual_learning)
 
-                should_turn_off_mutual_learning = self._should_turn_off_mutual_learning(self.epoch)
-                coeff_mutual_learning = int(not should_turn_off_mutual_learning)
+                    loss += coeff_mutual_learning * mutual_loss / (num_models - 1)
```
Comment (sovrasov) [inline torchreid/engine/image/am_softmax.py]: Why do we need to compute the mutual loss if `should_turn_off_mutual_learning` is `False`?

**Comment by sovrasov** [comment]
Also we now could remove `ext_modules` from setup.py. This would allow avoiding compilation of C code.

File: requirements.txt (hunk 1 above)
Comment (kprokofi) [reply requirements.txt]: Forgot to change this back

**Comment by sovrasov** [comment]
The target branch was updated to the latest `ote` state

File: torchreid/engine/image/am_softmax.py (hunk 6 above)
Comment (kprokofi) [reply torchreid/engine/image/am_softmax.py]: Now, it is seems to be okay

**Review by sovrasov — APPROVED** [review]
